test/functional-api/conversions/conversion.it-spec.ts

```diff
@@ -0,0 +1,405 @@
+    // console.log(chalDataLeadOrg);
+    //console.log(newHubDataNameId);
+    // console.log(chalDataNameId);
+
+    console.log(newHubDataCommunityOpp);
+    console.log(oppDataCommunity);
+
+    const newHubId = res.body.data.convertChallengeToHub.id;
+    const newChallengeId = res.body.data.convertChallengeToHub.challenges[0].id;
+
+    // expect(newHubDataCommunity).toEqual(chalDataCommunity); - fails with COMMUNITY_JOIN is missing after conversion
+    // expect(newHubDataPreferences).toEqual(chalDataPreferences); - fails - different preferences on different entities (to be verified manually)
+    // expect(newHubDataTagset).toEqual(chalDataTagset); - fails - tags are not converted
+    // expect(newHubDataTemplates).toEqual(chalDataTemplates); - challenges doesn't have templates
+    // expect(newHubDataApplication).toEqual(chalDataApplication);  excluded as applications are asserted as part of community.applications
+
+    // assert converted challenge with old challenge
+    expect(newHubDataContext).toEqual(chalDataContext);
+    expect(newHubDataAgent).toEqual(chalDataAgent);
+    expect(newHubDataAuthorization).toEqual(chalDataAuthorization);
+    //expect(newHubDataChallenges).toEqual(chalDataChallenges);
+    expect(newHubDataOpportunities).toEqual(chalDataOpportunities);
+    expect([newHubDataHost]).toEqual(chalDataLeadOrg);
+    expect(newHubDataNameId).toEqual(chalDataNameId);
+    expect(newHubDataDisplayName).toEqual(chalDataDisplayName);
+
+    // assert converted opportunity with old opportunity
+    // expect(newHubDataCommunityOpp).toEqual(oppDataCommunity);
+    expect(newHubDataContextOpp).toEqual(oppDataContext);
+    expect(newHubDataAgentOpp).toEqual(oppDataAgent);
+    expect(newHubDataAuthorizationOpp).toEqual(oppDataAuthorization);
+    //expect(newHubDataChallenges).toEqual(chalDataChallenges);
+    expect(newHubDataOpportunitiesOpp).toEqual([]);
+    // expect([newHubDataHostOpp]).toEqual(oppDataLeadOrg);
+    // expect(newHubDataNameIdOpp).toEqual(oppDataNameId); // changed nameId after conversion
+    expect(newHubDataDisplayNameOpp).toEqual(oppDataDisplayName);
+
+    await removeChallenge(newChallengeId);
+    await removeHub(newHubId);
+  });
+
+  test('Convert Challenge with 1 lead Organization to Hub', async () => {
+    const resCh = await mutation(
+      createChallenge,
+      challengeVariablesData(
+        challengeName,
+        `success-chnameid${uniqueId}`,
+        entitiesId.hubId
+      )
+    );
+
+    const newChallId = resCh.body.data.createChallenge.id;
+    const newChCommunityId = resCh.body.data.createChallenge.community.id;
+    const h = await assignOrganizationAsCommunityLeadFunc(
+      newChCommunityId,
+      entitiesId.organizationId
+    );
+
+    await assignUserAsCommunityMemberFunc(newChCommunityId, users.hubMemberId);
+    await assignUserAsCommunityLeadFunc(newChCommunityId, users.hubMemberId);
+    const chalRes = await getChallengeData(entitiesId.hubId, newChallId);
+
```

Comment:
Please save chalRes.body.data.hub.challenge in a separate variable so that latter code becomes more readable.